pkg/model/iam/iam_builder.go

```diff
@@ -571,13 +602,22 @@ func addECRPermissions(p *Policy) {
 }
 
 func (b *PolicyBuilder) addRoute53Permissions(p *Policy) {
```

Comment:
I think this should be called `addDNSControllerPermissions()`. What needs the permissions is more important than what the permissions give access to.

Reply:
I like that distinction that we can make now :-)